daemon/cmd/cells.go

```diff
@@ -208,6 +215,15 @@ var (
 		// ServiceCache holds the list of known services correlated with the matching endpoints.
 		k8s.ServiceCacheCell,
 
+		// K8s policy resource watcher cell. It depends on the daemon which we cast into
+		// the interface type here to avoid a circular package import
+		cell.Provide(func(p promise.Promise[*Daemon]) promise.Promise[policyK8s.PolicyManager] {
```

Comment:
Do you also have a plan in mind for how to factor out the PolicyManager things out of the daemon, so that this isn't needed anymore?

Reply:
It is on our radar (c.f. https://github.com/cilium/cilium/issues/30162) but it's a lower priority for us at the moment.